performance-tests/bench_2/builder/Topic.cpp

```diff
@@ -83,6 +83,7 @@ Topic::Topic(const TopicConfig& config, DDS::DomainParticipant_var& participant,
     throw std::runtime_error(ss.str());
   }
 
+#ifndef OPENDDS_NO_CONTENT_FILTERED_TOPIC
   for (unsigned int i = 0; i != content_filtered_topics_.length(); ++i) {
     DDS::ContentFilteredTopic_var cft =
       participant_->create_contentfilteredtopic(content_filtered_topics_[i].cft_name,
```

Comment:
It dawns on me that we're not currently supporting delayed enable() (based the autoenable_created_entities QoS policy being set to false). I think the easiest thing to do here is to call enable all content filtered topics for the current topic in Topic::enable().

Reply:
I am getting this error:

error C2039: 'enable': is not a member of 'DDS::ContentFilteredTopic'
Does ContentFilteredTopic have its own enable function?

Reply:
Only Entities can be enabled.  CFT is not an Entity.

Reply:
Ah, I assumed they were still entities. Nevermind!